metagraph/src/cli/build.cpp

```diff
@@ -107,7 +107,7 @@ int build_graph(Config *config) {
 
             auto constructor = boss::IBOSSChunkConstructor::initialize(
                 boss_graph->get_k(),
-                config->canonical,
+                config->graph_mode == DeBruijnGraph::CANONICAL,
```

Comment:
why not pass `graph_mode` directly to `IBOSSChunkConstructor` and let it handle what to do with the mode (rather than losing precision here)

Reply:
Because it doesn't make sense. There are just 2 options. Why should we pass a thing which has 3 states?